### DISCUSSION TITLE
联合制作的跨启动器联机功能中增加控制台

### DISCUSSION THREAD

**Discussion by turuihang** [Meloong-Git/PCL#822] [提问] [answered]
方便玩家对联机服务器进行控制及调整，有利于玩家的使用
样例：https://aternos.org/console/

**Comment by zkitefly** [comment, marked as answer]
好像声明过新联机不会弄服务器相关的适配，只为客户端与客户端的联机

**Comment by Pigeon0v0** [comment]
按照道理 Octo 就是一个做 P2P 打洞的库...（甚至 UPnP 都需要启动器自己映射）

只要启动器高兴就可以检测本地服务端开放的端口，但既然需要手动配置服务端，那么你应该能在本地 cmd 窗口看到 Console
